### ISSUE TITLE
Fixed Axis on Scrollable Chart

### ISSUE THREAD

**Issue by roly151** [infinum/flutter-charts#33]
The values on the horizontal axis for a fixed axis scrollable chart are cut off. Only 1 digit shows. This happens in the example app. How we I ensure that the horizontal axis shows the full value? 
![Screen Shot 2021-12-17 at 5 03 52 pm](https://user-images.githubusercontent.com/27068792/146497442-eb2b8cb0-be08-4e7f-a37c-54da3c761cc8.png)
p

**Comment by lukaknezic** [comment]
Hmm that should be fixed. I'll check it out. Thanks for reporting 😃 

@roly151 Just to check, it works fine if you use regular axis in scrollable chart (without fixed axis)?

**Comment by roly151** [comment]
@lukaknezic Yes, it works fine if you use regular axis in a scrollable chart

**Comment by irshadshan000** [comment]
Hi @lukaknezic ,

**Comment by lukaknezic** [comment]
Hey @irshadshan000,

Sorry for the delay. Yeah It looks like I accidentally clipped the text in the decoration. Fix will be merged for next release

**Comment by irshadshan000** [comment]
Yeah sure, Waiting for more features in Scrollable Chart,  thank you..
